sympy/physics/tests/test_units.py

```diff
@@ -20,6 +20,8 @@ def test_units():
     t = Symbol('t')
     assert integrate(t*m/s, (t, 1*s, 5*s)) == 12*m*s
     assert (t * m/s).integrate((t, 1*s, 5*s)) == 12*m*s
+    
+    assert float( (year / week).evalf(15) ) == 52.17745571428571
```

Comment:
If you compare Python floats you must use the `abs(act - exp) < tol`. But if you compare SymPy `Float`s, which are arbitrary precision, then you can compare all the digits: `(year / week).evalf(15) == S(52.17745571428571)`. (I think).